adapters/beachfront/beachfront.go

```diff
@@ -397,10 +481,23 @@ func impsToSlots(imps []openrtb.Imp, errs []error) (bannerRequest, int8, []error
 		}
 
 		if len(slot.Sizes) == 0 {
-			slot.Sizes = append(slot.Sizes, size{
-				W: *imp.Banner.W,
-				H: *imp.Banner.H,
-			})
+			// *imp.Banner.W == nil is not valid. I have to dereference to make this comparison. Also, this does
+			// get caught in /endpoints/openrtb2/auction.go L441 when I send in the request from test case
+			// "banner-no-size-nose-dive", but it gets through on that test case.
+
+			impDeref := &imp
```

Comment:
I don't think you need `impDeref` here. just checking `if imp.Banner.W == nil || imp.Banner.H == nil` should work. You are already doing a dereferencing in the `*imp.Banner.W` construction, this `impDeref` is a pointer to the imp structure that is then implicitly dereferenced in your code.

Reply:
I don't need it. Removed locally. I had copy pasted the "*imp.Banner.W" from a few lines below and gotten an error that I couldn't compare that with nil, and I can't. Just needed to take the "*" off. Dumb mistake.